src/all/komga/src/eu/kanade/tachiyomi/extension/all/komga/KomgaFilters.kt

```diff
@@ -0,0 +1,375 @@
+package eu.kanade.tachiyomi.extension.all.komga
+
+import android.content.SharedPreferences
+import eu.kanade.tachiyomi.network.GET
```

Comment:
```suggestion
import android.content.SharedPreferences
import eu.kanade.tachiyomi.extension.all.komga.dto.AuthorDto
import eu.kanade.tachiyomi.extension.all.komga.dto.CollectionDto
import eu.kanade.tachiyomi.extension.all.komga.dto.LibraryDto
import eu.kanade.tachiyomi.network.GET
```

the build was [failing](https://github.com/tachiyomiorg/tachiyomi-extensions/actions/runs/3407399293/jobs/5667024436#step:4:877) because of the missing imports after moving DTOs